CHANGELOG.md

```diff
@@ -58,13 +58,11 @@
 - Bump AppCompat to v1.4.1
 - Update the highlight alignment for newly added NHID or BP passport
 - Add highlight when newly scanned bp passports are added
-- Add support for demo country
 - Index `patientUuid` in tables that are joined in `OverdueAppointment` database view
 
 ### Changes
 
-- Implement adding scan support for India NHID in `EditPatientScreen`
-- Implement adding scan support for BpPassport in `EditPatientScreen`
+- Add health ID's from edit patient screen
 
 ## On Demo
```

Comment:
@msasikanth can we change this on Demo to the release date and number?

Reply:
Ah right, I made the release after I opened the PR. My bad.